archetypes/README.md

```diff
@@ -46,6 +46,7 @@ mvn archetype:generate -DarchetypeGroupId=org.apache.servicecomb.archetypes -Dar
 In console Interactive mode, input your GroupId, ArtifactId and Version of new project(module), after a while the new project will be generated.
 
 *Notice: We will publish these archetypes to maven center repository since 1.0.0-m2, if you would like to use an archetype from an unreleased version : *
+
 ```bash
-mvn archetype:generate -DarchetypeGroupId=org.apache.servicecomb.archetypes -DarchetypeArtifactId=business-service-jaxrs-archetype -DarchetypeVersion=${archetype-version} -DarchetypeRepository=https://repository.apache.org/content/groups/snapshots-group
+mvn org.apache.maven.plugins:maven-archetype-plugin:2.4:generate -DarchetypeGroupId=org.apache.servicecomb.archetypes -DarchetypeArtifactId=business-service-jaxrs-archetype -DarchetypeVersion=1.0.0-m2-SNAPSHOT -DarchetypeRepository=https://repository.apache.org/content/groups/snapshots-group
```

Comment:
by the old command, i succeeed to create project and run "mvn install"
(just copy and run, did no change anything, even did not replace ${archetype-version} to be 1.0.0-m2-SNAPSHOT)
so why we need to do this change?

and maybe this is a problem:
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ aid ---
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding GBK, i.e. build is platform dependent!
[INFO] Compiling 3 source files to D:\aid\target\classes

Reply:
mvn archetype:generate may use latest version 3.0+ of archetype-plugin,archetypeRepository had deleted,https://maven.apache.org/archetype/maven-archetype-plugin/generate-mojo.html

Reply:
ok, i see

but maybe encoding is a problem.

Reply:
In the release version we do need to specify the archetypeRepository. 
I think it's better explain -DarchetypeXXX options to avoid making user confused.

Reply:
Done